app/components/GameInstructions/GameInstructions.js

```diff
@@ -42,25 +43,18 @@ export default function Instructions({ instructionsState, }) {
       addInstruction(goBackToToggleTileMode);
     }
 
-    function createInstructionElements(arrayOfInstructionElements, instruction, index) {
-      if (index > 0) {
-        arrayOfInstructionElements.push(<h6 key={ `connector${index}` }>OR</h6>);
-      }
-      arrayOfInstructionElements.push(<h2 key={ `instruction${index}` }>{instruction}</h2>);
-      return arrayOfInstructionElements;
-    }
-
-    return instructionsList.reduce(createInstructionElements, []);
+    return instructionsList;
   }
 
   const currentInstructions = createInstructions(instructionsState);
+  const instructionsProps = { instructions: currentInstructions, };
   return (
     <div>
-      { currentInstructions }
+      <InstructionsBox { ...instructionsProps } />
```

Comment:
When you've only got a prop or two, extracting them only adds cognitive load instead of reducing it. I'd inline it.

Reply:
done

Frankly, I was trying to keep a unified style - so that always are some kind of props passed into an element via `{ ...*some*Props }`

but I do agree that if there are is only one prop, no point on making things more complicated